playwrighte2e/pages/jurisdictionPage.ts

```diff
@@ -0,0 +1,25 @@
+import { BasePage } from "./basePage";
+import { expect } from "@playwright/test";
+
+
+export default class JurisdictionPage extends BasePage {
```

Comment:
Is it worth renaming this page to BFActionsPage as it's doing BFs rather than jurisdictions

Reply:
Seems to be the same as playwrighte2e/pages/bfActionPage.ts